src/main/java/com/openshift/jenkins/plugins/pipeline/Auth.java

```diff
@@ -148,9 +157,47 @@ public boolean allowCertificate(final X509Certificate[] certificateChain) {
 
 	@Override
 	public boolean allowHostname(String hostname, SSLSession sslSession) {
```

Comment:
is this called for every connection, or just ones whose certs which fail standard hostname verification? is this implementing https://docs.oracle.com/javase/7/docs/api/javax/net/ssl/HostnameVerifier.html?